components/dashboard/src/Login.tsx

```diff
@@ -170,72 +175,79 @@ export const Login: FC<LoginProps> = ({ onLoggedIn }) => {
                     id="login-section-column"
                     className={"flex-grow max-w-2xl flex flex-col h-100 mx-auto" + (showWelcome ? " lg:my-0" : "")}
                 >
-                    <div className="flex-grow h-100 flex flex-row items-center justify-center">
-                        <div className="rounded-xl px-10 py-10 mx-auto">
-                            <div className="mx-auto pb-8">
-                                <img
-                                    src={providerFromContext ? gitpod : gitpodIcon}
-                                    className="h-14 mx-auto block dark:hidden"
-                                    alt="Gitpod's logo"
-                                />
-                                <img
-                                    src={providerFromContext ? gitpodDark : gitpodIcon}
-                                    className="h-14 hidden mx-auto dark:block"
-                                    alt="Gitpod dark theme logo"
-                                />
-                            </div>
+                    {needsSetupCheckLoading ? (
```

Comment:
This diff is gnarlier than the change was. Basically shifted previous content behind 2 checks that may render something else.